lib/include/openamp/virtio.h

```diff
@@ -152,7 +152,7 @@ struct virtio_device_id {
 	sizeof(struct vring_used_elem) * n + sizeof(uint16_t))
 
 #define VRING_DECLARE(name, n, align) \
-static char __vrbuf_##name[VIRTIO_RING_SIZE(n, align)] __aligned(VRING_ALIGNMENT); \
+static char __vrbuf_##name[VIRTIO_RING_SIZE(n, align)] __aligned(VIRTIO_AMP_VRING_ALIGNMENT); \
```

Comment:
If i well understood here you expect to reserve vrings memory with a `VIRTIO_AMP_VRING_ALIGNMENT` Alignement. to guaranty that a 4K page is reserved for one vring.

What about  using a parammeter instead of hardcoding it? 
And how do you ensure that the host is aligned with the __vrbuf_##name address?
As there is not usage yet of this macro, I would be in favor of removing it and reintroduce it  when used.
else we should maintain it evenif not usable in a generic way.

Reply:
I have removed the define in the newer version of the PR.